tdesign/src/Icon/cssSvgStrings.ts

```diff
@@ -28,6 +28,8 @@ export const apiIconURI =
   "data:image/svg+xml;charset=utf8,%3Csvg width='24' height='24' viewBox='0 0 24 24' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M13.5 3L10 20' stroke='black'/%3E%3Cpath d='M7.89996 8L3.49996 12L7.89996 16' stroke='%23000202' stroke-linejoin='round'/%3E%3Cpath d='M16 8L20.4 12L16 16' stroke='%23000202' stroke-linejoin='round'/%3E%3C/svg%3E";
 export const arrowRightURI =
   "data:image/svg+xml;charset=utf8,%3Csvg width='10' height='10' viewBox='0 0 10 10' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M7.4 5L7.90451 5.55495C8.06086 5.41281 8.15 5.21131 8.15 5C8.15 4.78869 8.06086 4.58719 7.90451 4.44504L7.4 5ZM3.5045 9.55496L7.90451 5.55495L6.8955 4.44504L2.4955 8.44504L3.5045 9.55496ZM7.90451 4.44504L3.5045 0.445044L2.49549 1.55495L6.8955 5.55495L7.90451 4.44504ZM7.4 4.25L-1.59301e-07 4.25L-1.9039e-07 5.75L7.4 5.75L7.4 4.25Z' fill='%23F94569'/%3E%3C/svg%3E";
+export const arrowRight2URI =
```

Comment:
Quick question because I don't know what did we decide - are you going to create a new icon component for this icon?

If not now, is there a CU task to create it?

Reply:
For reference for anyone reading this MR, this is how the icon looks like :smile: 

![image](https://user-images.githubusercontent.com/889383/142605834-dc13b6ba-5551-4ac4-af6f-80784deeb412.png)

Reply:
I renamed `arrowRight2URI` to `arrowRightLargeURI` to better reflect how it diifers from the original `aarowRightURI`. I was unable to get figure out why I could not properly scale the original `arrowRightURI` even though it is an SVG